lib/src/container.dart

```diff
@@ -323,6 +323,7 @@ class Authgear implements AuthgearHttpClientDelegate {
     );
   }
 
+  /// [state]: OAuth 2.0 state value
```

Comment:
You should remove `String? state` in line 333, instead of adding documentation for it.

Reply:
i saw the issue stated `For cross platform SDKs, describe it as OAuth 2.0 state value` for `state`
maybe thats outdated requirement?
let me remove this👌

Reply:
removed
<img width="532" alt="Screenshot 2024-09-05 at 2 24 37 PM" src="https://github.com/user-attachments/assets/41fb0943-9132-4ccc-b2fe-3196bb6c511d">